### PR TITLE
feat(integer): add default signed_scalar div/rem/div_rem

### PR DESCRIPTION
**Pull request by tmontaigu**
### Check-list:

* [x] Tests for the changes have been added (for bug fixes / features)

### CODE REVIEW

**Comment by tmontaigu** [comment]
@slab-ci cpu_fast_test

File: tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs [hunk 1]
```diff
@@ -565,6 +566,115 @@ impl ServerKey {
...
+
+        *numerator = self.unchecked_signed_scalar_div_parallelized(numerator, divisor);
+    }
+
+    /// Computes and returns the quotient of the division between
+    /// a signed ciphertext and a signed clear value.
+    ///
+    /// # Note
+    ///
+    /// - This division rounds (truncates) the quotient towards 0    /// # Note
```
Comment (IceTDrinker) [inline tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs:620]: /// # Note

floating at the end here

File: tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs [hunk 2]
```diff
@@ -565,6 +566,115 @@ impl ServerKey {
...
+    ) where
+        T: SignedReciprocable + ScalarMultiplier,
+        <<T as SignedReciprocable>::Unsigned as Reciprocable>::DoublePrecision: Send,
+    {
+        if !numerator.block_carries_are_empty() {
+            self.full_propagate_parallelized(numerator)
+        }
+
+        let (_quotient, remainder) =
+            self.unchecked_signed_scalar_div_rem_parallelized(numerator, divisor);
```
Comment (IceTDrinker) [inline tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs:674]: no unchecked rem parallelized ? not a blocker

**Review by IceTDrinker — COMMENTED** [review]
Looks good apart from that small error in a docstring

File: tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs (hunk 2 above)
Comment (tmontaigu) [reply tfhe/src/integer/server_key/radix_parallel/scalar_div_mod.rs]: added it as its a simple call to div_rem

**Review by IceTDrinker — APPROVED** [review]

**Review by IceTDrinker — APPROVED** [review]

**Pull request merged**